modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/helper/ViewExtensions.kt

```diff
@@ -0,0 +1,58 @@
+package au.com.shiftyjelly.pocketcasts.views.helper
+
+import android.graphics.Outline
+import android.view.View
+import android.view.ViewOutlineProvider
+import android.widget.TextView
+import au.com.shiftyjelly.pocketcasts.localization.helper.TimeHelper
+import au.com.shiftyjelly.pocketcasts.models.entity.BaseEpisode
+import au.com.shiftyjelly.pocketcasts.models.entity.UserEpisode
+import au.com.shiftyjelly.pocketcasts.models.type.UserEpisodeServerStatus
+import au.com.shiftyjelly.pocketcasts.utils.extensions.toLocalizedFormatLongStyle
+import java.util.Date
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+object ViewExtensions {
```

Comment:
Can you remove the outer object? I don't think it serves any purpose.

Reply:
Sure - I did the same for `player` module as well